src/scss/06_components/paragraphs/_event-list.scss

```diff
@@ -0,0 +1,181 @@
+.component--event-list .event {
+  border-bottom: 1px solid #e5e5e5;
+  padding-bottom: $spacing-half;
+  padding-top: $spacing-and-half;
+
+  @include breakpoint($breakpoint-l) {
+    padding-bottom: $spacing;
+    padding-top: $spacing;
+  }
+
+  &:first-child {
+    border-top: 1px solid #e5e5e5;
+  }
+
+  > a {
+    display: block;
+    text-decoration: none;
+
+    &:hover {
+      h3 {
+        text-decoration: underline;
+      }
+    }
+
+    @include breakpoint($breakpoint-l) {
+      display: grid;
+      grid-template-columns: 325px 1fr;
+    }
+  }
+
+  h3 {
+    margin-bottom: $spacing-and-half;
+    margin-top: $spacing;
+  }
+
+  .event__image-container {
+    background-position: center center;
```

Comment:
This could be just background-position: center; If there's only one value specified the other one will always be center.